src/main/java/org/apache/sling/distribution/journal/kafka/KafkaEndpoint.java

```diff
@@ -36,4 +37,17 @@
     @AttributeDefinition(name = "Kafka Default API Timeout",
             description = "Kafka Default API Timeout in ms.")
     int kafkaDefaultApiTimeout() default 60000;
+    
+    @AttributeDefinition(name = "Security protocol",
+            description = "e.g. SASL_SSL")
+    SecurityProtocol securityProtocol() default SecurityProtocol.PLAINTEXT;
```

Comment:
How about using a `String` here ? It seems that the value is anyway resolved to a `String` field

Reply:
Makes sense